src/main/java/teammates/ui/template/CourseEditSectionRow.java

```diff
@@ -144,9 +144,15 @@ public ElementTag getToggleSessionLevelInSectionButton() {
             for (int j = 0; j < 3 && i + j < sectionNames.size(); j++) {
                 String name = Const.ParamsNames.INSTRUCTOR_SECTION_GROUP + sectionIndex
                               + Const.ParamsNames.INSTRUCTOR_SECTION + (i + j);
-                ElementTag checkbox = createCheckBox(sectionNames.get(i + j), name, sectionNames.get(i + j),
-                                                     i + j == sectionIndex);
-                specialSectionGroup.add(checkbox);
+                if (isSectionSpecial()) {
```

Comment:
Let's try to add a header comment to the method <code>createSpecialSectionsForSectionRow</code> because it is not descriptive or understandable at all.